manifest.json

```diff
@@ -136,38 +135,15 @@
       "images": [
         {
           "sharedTags": {
-            "1.0.7": {},
-            "1.0": {},
-            "lts": {}
-          },
-          "platforms": [
-            {
-              "dockerfile": "1.0/jessie/sdk",
-              "os": "linux",
-              "tags": {
-                "1.0.7-jessie": {}
-              }
-            },
-            {
-              "dockerfile": "1.0/nanoserver-sac2016/sdk",
-              "os": "windows",
-              "tags": {
-                "1.0.7-nanoserver-sac2016": {},
-                "1.0.7-nanoserver": {
-                  "isUndocumented": true
-                },
-                "1.0.7-nanoserver-$(nanoServerVersion)": {
-                  "isUndocumented": true
-                }
-              }
-            }
-          ]
-        },
-        {
-          "sharedTags": {
             "1.1.4": {},
             "1.1": {},
-            "1": {}
+            "1": {},
+            "1.0": {
```

Comment:
What are these new tags?

Reply:
Not new, just moved here as a part of condensing our duplicate SDK images.